tests/qos/test_buffer.py

```diff
@@ -955,12 +955,12 @@ def test_headroom_override(duthosts, rand_one_dut_hostname, conn_graph_facts, po
 
     original_speed = duthost.shell('redis-cli -n 4 hget "PORT|{}" speed'.format(port_to_test))['stdout']
     original_cable_len = duthost.shell('redis-cli -n 4 hget "CABLE_LENGTH|AZURE" {}'.format(port_to_test))['stdout']
-    original_profile = duthost.shell('redis-cli hget "BUFFER_PG_TABLE:{}:3-4" profile'.format(port_to_test))['stdout'][1:-1]
-    original_pg_size = duthost.shell('redis-cli hget "{}" size'.format(original_profile))['stdout']
+    original_profile = duthost.shell('redis-cli hget "BUFFER_PG_TABLE:{}:3-4" profile'.format(port_to_test))['stdout']
+    original_pg_size = duthost.shell('redis-cli hget "BUFFER_PROFILE_TABLE:{}" size'.format(original_profile))['stdout']
     original_pool_size = duthost.shell('redis-cli hget BUFFER_POOL_TABLE:ingress_lossless_pool size')['stdout']
     if DEFAULT_OVER_SUBSCRIBE_RATIO:
         original_shp_size = duthost.shell('redis-cli hget BUFFER_POOL_TABLE:ingress_lossless_pool xoff')['stdout']
-        original_pg_xoff = duthost.shell('redis-cli hget "{}" xoff'.format(original_profile))['stdout']
+        original_pg_xoff = duthost.shell('redis-cli hget "BUFFER_PROFILE_TABLE:{}" xoff'.format(original_profile))['stdout']
     else:
         original_shp_size = None
         original_pg_xoff = None
```

Comment:
Line 1044 `check_pg_profile(duthost, 'BUFFER_PG_TABLE:{}:3-4'.format(port_to_test), original_profile.split(':')[1])`
should be `check_pg_profile(duthost, 'BUFFER_PG_TABLE:{}:3-4'.format(port_to_test), original_profile`.
Originally, `original_profile` was prefixed with `BUFFER_PROFILE_TABLE:` so we needed to split it. Now that the table name has been removed, we don't need to split it.

Reply:
@stephenxs  Addressed issue

Reply:
As commented above, the fix is no longer necessary as the table name is always in `original_profile`